android/java/org/chromium/chrome/browser/preferences/BravePreferenceKeys.java

```diff
@@ -6,6 +6,6 @@
  package org.chromium.chrome.browser.preferences;
  
  public final class BravePreferenceKeys {
-    public static final String BRAVE_BOTTOM_TOOLBAR_SET_KEY = "brave_bottom_toolbar_set";
-    public static final String BRAVE_BOTTOM_TOOLBAR_ENABLED_KEY = "brave_bottom_toolbar_enabled";
+    public static final String BRAVE_BOTTOM_TOOLBAR_ENABLED_KEY = "brave_bottom_toolbar_enabled_key";
+    public static final String BRAVE_BOTTOM_TOOLBAR_SET_KEY = "brave_bottom_toolbar_enabled";
```

Comment:
we mixed preferences names initially on feature migration. We kept one settings named the same but it was used for diff things comparing to tabs and another settings I just renamed to be consistent with the preference name. It breaks beta build settings for bottom toolbar but otherwise we don't have a way to migrate our existing stable version users.